frappe/utils/safe_exec.py

```diff
@@ -291,6 +292,7 @@ def get_safe_globals():
 		run_script=run_script,
 		is_job_queued=is_job_queued,
 		get_visible_columns=get_visible_columns,
+		get_content_hash=frappe.utils.data.sha256_hash,
```

Comment:
Change the name also? 
```suggestion
		sha256_hash=frappe.utils.data.sha256_hash,
```

Also utils are loaded separately in safe_exec. See the list of whitelisted utils in same file.